src/validationinterface.cpp

```diff
@@ -170,7 +170,7 @@ void ThreadNotifyWallets(CBlockIndex *pindexLastTip)
                 blockStack.emplace_back(
                     pindex,
                     oldFrontiers,
-                    recentlyConflicted.first.at(pindex));
+                    recentlyConflicted.first[pindex]);
```

Comment:
What's the reason for this change?

(`.at` checks whether the key exists in the collection and raises an exception if not; `operator[]` has undefined behaviour in that case.)

Reply:
`operator[]` does not have undefined behavior on this type (`std::map`); it [inserts the default value if the key doesn't exist](https://en.cppreference.com/w/cpp/container/map/operator_at). Here this would be an empty `std::vector` which has the semantics we want (no recently-conflicted transactions).